src/diff.js

```diff
@@ -31,7 +31,9 @@ module.exports.getDiffMapping = async ( pushConfig, number, github ) => {
 			}
 			chunk.changes.forEach( change => {
 				position++;
-				mapping[ file.to ][ change.ln || change.ln2 ] = position;
+				if( change.type === "add" ) {
```

Comment:
```suggestion
				if ( change.type === "add" ) {
```

What about for removals? 🤔

Reply:
we don't have to review the code that has been removed in terms of security/ performance issues.

Reply:
Hmm, true; I thought it might screw up the offsetting or not work for changes, but seems fine.